Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "dioxus-heroicons"
-version = "0.1.4"
+version = "2.0.13"
```

Comment:
I don't think it makes sense to change the crate version to match the heroicons version, at least not right now. The crate has only had a few releases, so I don't want to give it a stable version number yet.